q2_feature_table/_summarize/tabulate_seqs_assets/index.html

```diff
@@ -9,7 +9,7 @@
         button on the resulting page.
       <p>
         To download a raw FASTA file of your sequences, click
-        <a href="sequences.fasta" download="sequences.fasta">here</a>.
+        <a href="sequences.fasta" target="_blank">here</a>.
```

Comment:
Oops, submitted with no description. Someone could make a shady fasta file 🕵️ , might want to include that `rel="noopener noreferrer"` to any `target="_blank"` attrs across all plugins.

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/